apps/services/bff/src/app/constants/time.ts

```diff
@@ -0,0 +1,8 @@
+export const FIVE_SECONDS_IN_MS = 5 * 1000
+export const ONE_HOUR_IN_MS = 60 * 60 * 1000
```

Comment:
Let's have this configurable in environment/BffConfig, eg "REFRESH_TOKEN_INACTIVE_LIFETIME" or "BFF_SESSION_DURATION"

Reply:
Moved to config but I am keeping FIVE_SECONDS_IN_MS since it being used in two places